newIDE/app/src/ObjectsRendering/Renderers/RenderedInstanceWithCustomizableCenter.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import RenderedInstance from './RenderedInstance';
+
+/**
+ * RenderedInstance is the base class used for creating renderers of instances,
+ * which display on the scene editor, using Pixi.js, the instance of an object (see InstancesEditor).
+ */
+export default class RenderedInstanceWithCustomizableCenter extends RenderedInstance {
+  _isCenterCustomizable = true;
```

Comment:
Seems useless?

Reply:
Oh yes, I though I could use this instance of `renderedInstance instanceof RenderedInstanceWithCustomizableCenter`. I'll remove it